locust/locustfile.py

```diff
@@ -48,38 +41,33 @@ def check_count(self):
     def record_req(self):
         self.count += 1
 
+    def get_token(self):
+        secure_random = random.SystemRandom()
+        return secure_random.choice(self.tokens)
+
+    def set_auth_header(self, token):
+        self.client.headers['Authorization'] = 'Bearer ' + token
+
+    def get_random_userinfo(self):
+        access_token = self.get_token()
+        self.set_auth_header(access_token)
+        response = self.client.get("%s%s" % (base_url, '/v1/connect/userinfo'))
+        data = json.loads(response.content)
+        return data.get('patient')
+
     def make_req(self, resource):
         self.check_count()
         self.client.get("%s%s" % (base_url, resource))
         self.record_req()
 
-    @task(11)
-    def get_userinfo(self):
-        resource = '/v1/connect/userinfo'
-        self.make_req(resource)
-
-    @task(26)
+    @task()
     def get_eob(self):
-        resource = '/v1/fhir/ExplanationOfBenefit/?patient=%s' % self.patient
-        self.make_req(resource)
-
-    @task(13)
-    def get_coverage(self):
-        resource = '/v1/fhir/Coverage/'
-        self.make_req(resource)
-
-    @task(22)
-    def get_patient(self):
-        resource ='/v1/fhir/Patient/%s' % self.patient
-        self.make_req(resource)
-
-    @task(6)
-    def get_metadata(self):
-        resource = '/v1/fhir/metadata'
+        patient = self.get_random_userinfo()
+        resource = '/v1/fhir/ExplanationOfBenefit/?patient=%s' % patient
         self.make_req(resource)
```

Comment:
at the end of the sequence (which I think is here?) can you make a request to the `revoke-token` endpoint so the load test can kind of clean up after itself?